examples/case_studies/bayesian_ab_testing_introduction.ipynb

```diff
@@ -1,5 +1,20 @@
 {
```

Comment:
<p>Can you also add a watermark section title right above this code cell? As a level 2 title like the authors and references ones</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>